tools/spades_header_fixer/spades_header_fixer.xml

```diff
@@ -0,0 +1,20 @@
+<tool id="spades_header_fixer" name="SPAdes fasta header fixer" version="1.1.1">
+	<description>Fixes Spades Fasta ids</description>
+	<command >sed -r "s/>NODE(_[0-9]+)_(.*)/>${input.name}\1 \2/g" $input >$output</command>
+	<inputs>
+		<param name="input" type="data" format="fasta" label="fix these fasta headers"/>
+		</inputs>
+	<outputs>
+		<data name="output" format="fasta"/>
+	</outputs>
+  <tests>
+    <test>
+         <param name="input" value="spades_before.fasta" />
+         <output name="output" file="spades_header_fixed.fasta" ftype="fasta"/>
+    </test>
+  </tests>
+
+	<help>
+	
```

Comment:
Can you add a one line description what the tool purpose is. 

Something along the lines of "Replacing default value of **NODE** from SPAdes fasta header(s) to filename or collection identifier"

Reply:
Added the description, looks something like this 
![image](https://user-images.githubusercontent.com/11945573/71120876-e34b4180-21a2-11ea-8d6c-6bd148dec873.png)